### PR TITLE
Fixup! for Invoke cct run before testsetup

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3273,14 +3273,16 @@ function onadmin_run_cct()
...
-    if [[ -n $cct_tests ]]; then
+    if iscloudver 6plus && [[ -n $cct_tests ]]; then
+
+        # - default checkout location for mkcloud based setups:
+        mkdir -p /root/github.com/SUSE-Cloud
+        pushd /root/github.com/SUSE-Cloud/
+        git clone https://github.com/SUSE-Cloud/cct.git
+        # TODO checkout branches for older versions
+        cd cct
+        bundle install
```
Comment (MaximilianMeister) [inline scripts/qa_crowbarsetup.sh]: (not introduced by you) but do you know if bundler installs system wide here?

**Comment by aspiers** [comment]
Huh?  This is barely different to #446 which is already merged.  Looks like you rewrote your branch without rebasing?

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (aspiers) [inline scripts/qa_crowbarsetup.sh]: Good question but this code is already merged via #446.  BTW see https://github.com/bundler/bundler/issues/3006 which just hit me (again) in a completely different way :(

Comment (MaximilianMeister) [inline scripts/qa_crowbarsetup.sh]: @aspiers the bundle install was even introduced earlier than https://github.com/SUSE-Cloud/automation/pull/446

so if it is installed system wide we should maybe restrict it to a local folder to not interfere with the other gems

**Comment by MaximilianMeister** [comment]
yes looks like not rebased... but then it should show a merge conflict here at least :-/

**Comment by aspiers** [comment]
No conflict because the new changes are independent of the old ones.

**Comment by dirkmueller** [comment]
@MaximilianMeister , @aspiers sorry, I was surprised that this got merged so quickly before I was able to finish testing.. :-)

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (vmoravec) [inline scripts/qa_crowbarsetup.sh]: It's not installed system wide, cct git repo has config file in `.bundle/config` for installing the gems into local `vendor/bundle` dir.

Comment (MaximilianMeister) [inline scripts/qa_crowbarsetup.sh]: @vmoravec ah good to know. thanks

**Comment by vuntz** [comment]
+1

**Comment by MaximilianMeister** [comment]
+1

**Pull request merged**